src/pages/Autolending/MainToggle.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-	<kv-toggle class="main-toggle" v-model="isEnabled">
+	<kv-toggle
+		class="main-toggle"
+		id="auto-lending-toggle"
+		v-model="isEnabled"
```

Comment:
Is v-model handled now? seems like it was removed in the new kv-toggle code.

Reply:
v-model is handled now with
```	
model: {
    prop: 'checked',
    event: 'change'
},
props: { ...
```
A little more info on that [here](https://vuejs.org/v2/guide/components-custom-events.html#Customizing-Component-v-model)

Reply:
Nice 👍